elements/lisk-validator/src/formatter.ts

```diff
@@ -121,3 +129,7 @@ validator.addKeyword('uniqueSignedPublicKeys', {
 	compile: () => (data: ReadonlyArray<string>) =>
 		new Set(data.map((key: string) => key.slice(1))).size === data.length,
 });
```

Comment:
The keyword `validator.addKeyword('uniqueSignedPublicKeys'` does not have a meta-schema. So completely unaware what it actually is what it accepts. Look at https://github.com/LiskHQ/lisk-sdk/blob/development/framework/src/controller/validator/keywords/arg/meta_schema.js for a sample. Also I don't think its a use case of a keyword, rather its just a validation format which requires uniqueness on keys.

Reply:
I think this is outside the scope of this PR for which only methods that were missing were added to this library. Please open an issue with suggestions for lisk-validator